src/jit/importer.cpp

```diff
@@ -1623,6 +1630,14 @@ GenTreePtr Compiler::impNormStructVal(GenTreePtr           structVal,
                 alreadyNormalized  = true;
             }
             else
+#endif
+#if FEATURE_HW_INTRINSICS
+                if (blockNode->OperGet() == GT_HWIntrinsic && blockNode->AsHWIntrinsic()->isSIMD())
```

Comment:
Nit: Indentation?

Reply:
All the JIT code formatted by clang-format.